diff --git a/node_modules/react-native-gifted-chat/lib/MessageText.js b/node_modules/react-native-gifted-chat/lib/MessageText.js
index 6f03145..eafea76 100644
--- a/node_modules/react-native-gifted-chat/lib/MessageText.js
+++ b/node_modules/react-native-gifted-chat/lib/MessageText.js
@@ -104,8 +104,8 @@ export default class MessageText extends React.Component {
             this.props.customTextStyle,
         ]} parse={[
             ...this.props.parsePatterns(linkStyle),
-            { type: 'url', style: linkStyle, onPress: this.onUrlPress },
-            { type: 'phone', style: linkStyle, onPress: this.onPhonePress },
+//             { type: 'url', style: linkStyle, onPress: this.onUrlPress },
+//             { type: 'phone', style: linkStyle, onPress: this.onPhonePress },
             { type: 'email', style: linkStyle, onPress: this.onEmailPress },
         ]} childrenProps={{ ...this.props.textProps }}>
           {this.props.currentMessage.text}
